packages/cli/src/commands/setup/auth/templates/clerk.auth.js.template

```diff
@@ -16,8 +16,10 @@ import { logger } from 'src/lib/logger'
  */
 export const getCurrentUser = async (
```

Comment:
Please rename this file from .js to .ts! Otherwise all hell will break loose!

Reply:
^^^⚠️ critical. Please do not merge before doing this.

Reply:
Done!